### PR TITLE
Account Inquiry - Pesalink Banks

### PR DESCRIPTION
**Pull request by kamalogudah**
## What is the Purpose?

This PR adds Account Inquiry - Pesalink Banks which returns the recipients’ Linked Banks linked to the provided phone number on PesaLink

## Issue(s) affected?

List of issues addressed by this PR.

### CODE REVIEW

**Review by lenileiro — APPROVED** [review]
🔥🔥

File: lib/ex_pesa/Jenga/send_money_queries/account_inquiry_pesalink_banks.ex [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      iex> ExPesa.Jenga.SendMoneyQueries.AccountInquiryPesalinkBanks.request(%{ mobileNumber: "0722000000"})
+      {:ok,
+        %{
+          "banks" => [
+            %{"bankCode" => "31", "bankName" => "Stanbic", "customerName" => "John Doe"}
+          ]
+        }
+      }
+  """
+  @spec request(map()) :: {:error, any()} | {:ok, any()}
```
Comment (TraceyOnim) [inline lib/ex_pesa/Jenga/send_money_queries/account_inquiry_pesalink_banks.ex:27]:
```suggestion
  @spec request(map()) :: {:error, any()} | {:ok, map()}
```

Comment (manuelgeek) [reply lib/ex_pesa/Jenga/send_money_queries/account_inquiry_pesalink_banks.ex]: I think if we look a the nature of responses, not all responses are always map, some are string with messages only @TraceyOnim

**Review by manuelgeek — APPROVED** [review]
looks good

**Review by lenileiro — APPROVED** [review]

**Pull request merged**